centinel/unit_test/invalid_hosts.txt

```diff
@@ -0,0 +1,5 @@
+www.googllss.com
+www.couchtuner.eu
+www.creattaaaiia.com
+www.beforeplusafter.com
+www.zigzagonly.com
```

Comment:
This is missing cases where the URL itself is malformed (e.g. contains spaces or other illegal characters).